.github/workflows/sync-mainnet.yml

```
@@ -47,10 +42,16 @@ jobs:
         if: ${{ failure() }}
         run: echo "GITHUB_RUN_STATE=1" >> $GITHUB_ENV
       - name: insert report to postgres
-        if: ${{ always() && secrets.PGHOST != "" }}
+        if: ${{ always() }}
         env:
           JOB_ID: "sync-mainnet-${{ github.repository }}-${{ github.ref_name }}-${{ steps.date.outputs.date }}-in-10h"
-        run: ./ckb-sync-mainnet/script/sync-mainnet.sh insert_report_to_postgres
+          PGHOST: ${{ secrets.PGHOST }}
+          PGPORT: ${{ secrets.PGPORT }}
+          PGUSER: ${{ secrets.PGUSER }}
+          PGPASSWORD: ${{ secrets.PGPASSWORD }}
+          PGDATABASE: ckbtest
+        run: |
+            [ -z "${{ secrets.PGHOST }}" ] || ./ckb-sync-mainnet/script/sync-mainnet.sh insert_report_to_postgres
```

Since `env: PGHOST: ${{ secrets.PGHOST }}`, the follow code is enough:

```suggestion
            [ -z "${PGHOST}" ] || ./ckb-sync-mainnet/script/sync-mainnet.sh insert_report_to_postgres
```

---

can not using ${PGHOST} directly in yaml file , just can change `${{ secrets.PGHOST }}` to `${{ env.PGHOST }}`

---

[GitHub Docs: Example using Bash](https://docs.github.com/en/actions/security-guides/encrypted-secrets#example-using-bash):

```yaml
steps:
  - shell: bash
    env:
      SUPER_SECRET: ${{ secrets.SuperSecret }}
    run: |
      example-command "$SUPER_SECRET"
```